universal-login-contracts/contracts/ERC1077MasterCopy.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity ^0.5.2;
+
+import "./common/MasterCopy.sol";
+import "./ERC1077.sol";
+
+
+contract ERC1077MasterCopy is MasterCopy, ERC1077 {
```

Comment:
We should write a unit test with ERC1077 initiated as a master copy (with one key == `0x000...0`, make sure we can't execute any method.